docs/detections/add-exceptions.asciidoc

```diff
@@ -69,7 +69,9 @@ Add conditions that define when the exception prevents alerts:
 * Wildcards are not supported in value lists.
 * If a value list can't be used due to <<manage-value-lists,size or data type>>, it'll be unavailable in the *Value* menu.
 =======
-    * `matches` | `does not match` — Allows you to use wildcards in *Value*, such as `C:\path\*\app.exe`. Available wildcards are `?` (match one character) and `*` (match zero or more characters). The selected *Field* data type must be {ref}/keyword.html#keyword-field-type[keyword], {ref}/text.html#text-field-type[text], or {ref}/keyword.html#wildcard-field-type[wildcard].
+    * `matches` | `does not match` — Allows you to use wildcards in *Value*, such as `C:\\path\\*\\app.exe`. Available wildcards are `?` (match one character) and `*` (match zero or more characters). The selected *Field* data type must be {ref}/keyword.html#keyword-field-type[keyword], {ref}/text.html#text-field-type[text], or {ref}/keyword.html#wildcard-field-type[wildcard].
++
+NOTE: Some characters must be escaped with a backslash, such as `\"` for a quotation mark and `\\` for a literal backslash. Windows paths must be divided with double backslashes (for example, `C:\\Windows\\explorer.exe`), and paths that already include double backslashes might require four backslashes for each divider.      
```

Comment:
We don't believe quotes need to be escaped. Those will be escaped automatically in the backend. The characters that would need to be escaped explicitly that we know of are `\`, `*`, and `?`. If `*` is not escaped it would interpreted as `matches zero or more characters`. If `?` is not escaped it would be interpreted as `matched one character`.

Reply:
Oops, forgot to remove the first part about escaping the quotes. Let me know the following revision is accurate or needs additional changes:

```suggestion
NOTE: Some characters must be escaped with a backslash, such as `\\` for a literal backslash, `\* for an asterisk, and \? for a question mark. If `*` is not escaped, it is interpreted as `matches zero or more characters`. Similarly, if `?` is not escaped it is interpreted as "matched one character". Also note that Windows paths must be divided with double backslashes (for example, `C:\\Windows\\explorer.exe`), and paths that already include double backslashes might require four backslashes for each divider.      
```

Reply:
I agree, no need to duplicate the info. One more question: should the star char (`*`) be called the asterisk character or the wildcard character? I'm not sure which term is more accurate to use in the context of this sentence.

Reply:
I think that technically both `*` and `?` are considered "wildcard operators".

Reply:
In the context of this sentence, I think asterisk is correct because they would want to use `\*` if they want the value to be interpreted as an asterisk.